frontend/packages/kubevirt-plugin/integration-tests-cypress/view/dialog.ts

```diff
@@ -52,5 +55,49 @@ export const addDisk = (disk: Disk) => {
       .select(Cypress.env('STORAGE_CLASS'))
       .should('have.value', Cypress.env('STORAGE_CLASS'));
   }
+  if (disk.source === DISK_SOURCE.Url) {
```

Comment:
This should move to after line #32, and use a switch here maybe more readable.

Reply:
I'll fix it